### PR TITLE
feat: add nostrictsync option

### LINKED ISSUE #210: vhd disk mount parameters tuning
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

smb mount parameters:
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/162df829aa5f43094df59e1804a9cab0cac2b510/pkg/azurefile/nodeserver.go#L158

loop device mount parameters:
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/162df829aa5f43094df59e1804a9cab0cac2b510/pkg/azurefile/nodeserver.go#L200-L203
**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ZeroMagic**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
add `nostrictsync` option

**Which issue(s) this PR fixes**:

Fixes #210

**Special notes for your reviewer**:

**Release note**:
```

```

### CODE REVIEW

File: pkg/azurefile/nodeserver.go [hunk 1]
```diff
@@ -206,7 +206,7 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 		options := util.JoinMountOptions(mountFlags, []string{"loop"})
 		if strings.HasPrefix(fsType, "ext") {
 			// following mount options are only valid for ext2/ext3/ext4 file systems
-			options = util.JoinMountOptions(options, []string{"noatime", "barrier=0", "errors=remount-ro"})
+			options = util.JoinMountOptions(options, []string{"noatime", "barrier=1", "errors=remount-ro", "nostrictsync"})
```
Comment (andyzhangx) [inline pkg/azurefile/nodeserver.go:209]: add `nostrictsync` in L161, and keep `errors=remount-ro` here

Comment (ZeroMagic) [reply pkg/azurefile/nodeserver.go]: done

**Pull request merged**